bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestImport.java

```diff
@@ -43,22 +43,28 @@
   @Test
   @Category(KnownGap.class)
   public void testMapReduce() throws IOException, ClassNotFoundException, InterruptedException {
-    Table oldTable = getConnection().getTable(TABLE_NAME);
+    Admin admin = getConnection().getAdmin();
 
+    admin.disableTable(TABLE_NAME);
```

Comment:
call tableExists() before disableTable() and deleteTable()?
We should log and ignore the error if deletion fails.